assets/queries/k8s/k8s_dashboard_integration/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy [ result ] {
+   document := input.document
+   metadata := document[i].metadata
+   kind := document[i].kind
+   check_kind(kind)
+   
+   spec := input.document[i].spec
+   template_spec := spec.template.spec
```

Comment:
Use this auxiliary function to get the spec that can be used for all types of workloads (Deployment, Pod, ...):
```
getSpecInfo(document) = specInfo {
    templates := {"job_template", "jobTemplate"}
    spec := document.spec[templates[t]].spec.template.spec
    specInfo := {"spec": spec, "path": sprintf("spec.%s.spec.template.spec", [templates[t]])}
} else = specInfo {
    spec := document.spec.template.spec
    specInfo := {"spec": spec, "path": "spec.template.spec"}
} else = specInfo {
    spec := document.spec
    specInfo := {"spec": spec, "path": "spec"}
}
```
Replace this lines with `spec := getSpecInfo(document).spec`